.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,60 @@
+<!-- ----------------------------------------------------------------
+  If you're editing this as a result of an invocation of a GitHub CLI
+   tool, note that lines that begin with '#' are stripped. To preserve the
+   markdown that begins with '#' below, be sure to preserve the leading
+   whitespace on those lines.
```

Comment:
Interesting, I always end up deleting that leading white space since it always struck me as a typo.

Reply:
yeap, lots of folks did.  you're in good company